plugins/console/spec/consoleConfig.js

```diff
@@ -0,0 +1,17 @@
+var env = require('../../../spec/environment.js');
```

Comment:
It is possible!  If you look at the [tests for the `ngHint` plugin](https://github.com/angular/protractor/tree/master/plugins/ngHint/spec), you'll see both `failureConfig.js` and `successConfig.js`.  Then, in the `scripts/test.js`, you see that `successConfig.js` is pushed onto `passingTests`, but if you scroll down near the bottom you see

```js

// Check ngHint plugin

executor.addCommandlineTest(
    'node lib/cli.js plugins/ngHint/spec/failureConfig.js')
    .expectExitCode(1)
    .expectErrors([{
      message: 'warning -- ngHint plugin cannot be run as ngHint code was ' +
          'never included into the page'
    }, {
      message: 'warning -- ngHint is included on the page, but is not active ' +
          'because there is no `ng-hint` attribute present'
    }, {
      message: 'warning -- Module "xApp" was created but never loaded.'
    }]);
```
This basically means "run `failureConfig.js`, and expect it to fail with these errors"